packages/core/src/utilities/border/_border.docs.scss

```diff
@@ -9,14 +9,7 @@ The border utility class sets the `border-color` and `border-size`, with support
 
 @hide-markup
 
-Markup:
```

Comment:
ah yeah, the issue with stacking vertically is that it becomes a lot more verbose (allot more empty white space) and you lose the subgrouping (all borders, vertical, horizontal). I’m a bit divided on this, because if we add pixels we should add to everything, but this one kinda works without it, especially considering the pixels is just 1,2,3px in this case